test/test_job_archive_interface.py

```diff
@@ -108,64 +109,106 @@ def setUpClass(self):
     # passing a valid jobid should return
     # its job information
     def test_01_with_jobid_valid(self):
-        job_records = aclif.view_jobs_with_jobid(jobs_conn, 102, op)
+        args = Namespace(jobid=102)
+        job_records = aclif.view_job_records(jobs_conn, op, args)
```

Comment:
Rather than populating and passing a `Namespace` down into a function, I think the more re-usable method would be to have `view_job_records` have a `**kwargs` argument, and then you could call the function in a few different ways:
```
aclif.view_job_records(jobs_conn, op, **my_dict)
aclif.view_job_records(jobs_conn, op, user="102", after_start_time=5)
```

Which is nice from the testing perspective but also if anyone wants to call `view_job_records` directly in the future.